doc/module_examples/02_inspection_needed/check.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+. /usr/share/preupgrade/common.sh
+
+#END GENERATED SECTION
+
+#
+# The whole section above is processed and modified by preupg-xccdf-compose
+# script, according to content of INI file (in this case content.ini).
+# And in addition the LICENSE used by PreupgradeAssistant is inserted.
+#
+
```

Comment:
It is a nitpick, but part of the above section is also a shebang line. The wording as it is indicates that it is also provided by preupg-xccdf-compose. As far as I know it is not true as shebang line should be included right when writing module code (for good reasons). Please consider rewording to reflect that.